src/frontend/src/btc/components/send/BtcSendForm.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import BtcSendAmount from '$btc/components/send/BtcSendAmount.svelte';
+	import BtcSendDestination from '$btc/components/send/BtcSendDestination.svelte';
+	import type { BtcAmountAssertionError } from '$btc/types/btc-send';
+	import SendSource from '$lib/components/send/SendSource.svelte';
+	import ButtonGroup from '$lib/components/ui/ButtonGroup.svelte';
+	import ContentWithToolbar from '$lib/components/ui/ContentWithToolbar.svelte';
+	import { balance } from '$lib/derived/balances.derived';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { token } from '$lib/stores/token.store';
+	import type { NetworkId } from '$lib/types/network';
+
+	export let networkId: NetworkId | undefined = undefined;
+	export let amount: number | undefined = undefined;
+	export let destination = '';
+	export let source: string;
+
+	let amountError: BtcAmountAssertionError | undefined;
+	let invalidDestination: boolean;
+
+	const dispatch = createEventDispatcher();
+	const close = () => dispatch('icClose');
+</script>
+
+<form on:submit={() => dispatch('icNext')} method="POST">
+	<ContentWithToolbar>
+		<BtcSendDestination bind:destination bind:invalidDestination {networkId} on:icQRCodeScan />
+
+		<BtcSendAmount bind:amount bind:amountError />
+
+		<SendSource token={$token} balance={$balance} {source} />
+
+		<!--	TODO: calculate and display transaction fee	-->
+
+		<ButtonGroup slot="toolbar">
+			<button type="button" class="secondary block flex-1" on:click={close}
+				>{$i18n.core.text.cancel}</button
+			>
+			<button class="primary block flex-1" type="submit" disabled={false}>
+				{$i18n.core.text.next}
+			</button>
```

Comment:
For these two buttons, I think we already have some components

```svelte
<ButtonCancel on:click={close} />

<Button disabled={false}>
  {$i18n.core.text.next}
</Button>
```

But, for the cancel button, my suggestion is to use a slot

`<slot name="cancel" />`

like the other forms, because sometimes you will give a cancel button and sometimes a close button (see the other forms `IcSendFOrm` and `SendForm`)

Reply:
Good point, done.